### DISCUSSION TITLE
Basic Help

### DISCUSSION THREAD

**Discussion by reaperherpderp** [cyberofficial/Synthalingua#22] [Q&A] [answered]
im sorry, im new to all this lol. 
Ive downloaded and installed everything, which seems to have gone fine for now.
But now i need to know what to do to actually run the program so i can see if it is actually working.

**Comment by cyberofficial** [comment]
do you have a file called "livetranslation.bat" in the same folder where setup.bat is ??

**Reply by reaperherpderp** [reply]
i do not no
![Screenshot 2023-04-08 023521 - Copy](https://user-images.githubusercontent.com/130147516/230709860-44312f15-de4b-4ed8-b980-7a5f5960db2f.png)
![Screenshot 2023-04-08 022824 - Copy](https://user-images.githubusercontent.com/130147516/230709862-54820285-fc55-4bb7-913f-1ed5ef24bc61.png)

**Reply by cyberofficial** [reply]
after running set up, does it create the file for you?

**Comment by reaperherpderp** [comment]
2nd image was what it did when i ran the setup, and the 1st image was after i ran the setup and it did its downloads and install, so it doesnt look like it created it

**Reply by cyberofficial** [reply]
i notice the image says it can't find requirements.txt, did you extract the zip file?
also make sure you dont run as admin
as i see you installed it into your system32 folder meaning it was ran as admin

**Comment by reaperherpderp** [comment]
i had run the setup twice, the 2nd time as admin in case that was the issue.
i did extract the zip file, but i had in downloads still. I moved the file to C: drive and ran the set up again and it seems to have solved the problem i think
![Screenshot 2023-04-09 015338 - Copy](https://user-images.githubusercontent.com/130147516/230758990-72fe4a61-1def-4a1c-b74b-6baa65bf12c2.png)
where do i input the commands to change the parameters? i am reading the README.txt but it doesnt specify for an idiot like me, apologies for all the trouble i just dont want to mess anything up.

**Reply by cyberofficial** [reply, marked as answer]
Right click edit the live translation file, you can edit the arguments there.

**Reply by reaperherpderp** [reply]
ty for the help
